solutions/swb-ui/ui/end-to-end-tests/cypress/support/common-utils.ts

```diff
@@ -31,6 +35,7 @@ export function selectItemDropDown(componentTestId: string, itemsToSelect: strin
       .contains(new RegExp(itemRegex))
       .click();
   });
+  if (closeAfterSelection) cy.focused().blur();
```

Comment:
Hmm, why are we using blur? Couldn't we just click the drop down button again to close it?
`cy.get(`[data-testid="${componentTestId}"] button`).click();`